core/src/main/java/org/openstack4j/model/network/ext/builder/HealthMonitorV2Builder.java

```diff
@@ -49,11 +52,11 @@
 
     /**
      *
-     * @param poolId
-     *             The pool that this health monitor will monitor
+     * @param pools
+     *             List of pools that this health monitor will monitor
      * @return HealthMonitorV2Builder
      */
-    HealthMonitorV2Builder poolId(String poolId);
+    HealthMonitorV2Builder pools(List<LbPoolV2> pools);
```

Comment:
A lot of the v1 code stores these lists as strings ( https://github.com/ContainX/openstack4j/blob/master/core/src/main/java/org/openstack4j/model/network/ext/LbPool.java#L72 ). Should we be doing it this way? It's one thing I'm trying to figure out. 
In the json in v2 the objs are returnd like         ```
"pools": [
            {
                "id": "74aa2010-a59f-4d35-a436-60a6da882819"
            }
        ],
```
so it seems like maybe a list of strings isn't the best format to store things? thoughts?